### PR TITLE
created sales engineer outline issue 

### PR DESCRIPTION
**Pull request by devakisj1**
issue #650 resolved

### CODE REVIEW

File: roles-in-tech/sales-engineering.md [hunk 1]
```diff
@@ -2,91 +2,101 @@
...
 ### Objectives
 
 **Participants will be able to:**
 
-- Things
-- More things
-- Even more things
-- Even more things
+- Understand the role of a sales engineer in a company
+- Decide wheter they are suitable for a future as a sales engineer
```
Comment (manaswinidas) [inline roles-in-tech/sales-engineering.md]: whether in place of wheter

File: roles-in-tech/sales-engineering.md [hunk 2]
```diff
@@ -2,91 +2,101 @@
...
+- **Step 1: Earn a Bachelor's Degree**
+Many jobs for sales engineers require a bachelor's degree in engineering. Engineering programs contain required classes in math and science, as well as foundational engineering and general education courses.
+- **Step 2: Gain Sales Experience**
+After obtaining their engineering degree, it is important for new graduates to gain firsthand sales experience. This experience will allow them to advance to a sales engineer position as well as prepare them for licensure.
+- **Step 3: Consider Licensure**
+Some employers require their sales engineers to be professionally licensed.
+- **Step 4: Continue Your Education**
+ Sales engineers typically must continue their education to stay current with changing trends and products in their field. Additionally, re-training may be required as old technology becomes obsolete.
+
+
```
Comment (manaswinidas) [inline roles-in-tech/sales-engineering.md]: Please remove extra line .i.e, line 79

File: roles-in-tech/sales-engineering.md [hunk 3]
```diff
@@ -2,91 +2,101 @@
...
+- Go through [this article](https://www.careerexplorer.com/careers/sales-engineer/how-to-become/) for furthur infomation on how to become a SE.
```
Comment (manaswinidas) [inline roles-in-tech/sales-engineering.md]: information in place of infomation and further in place of furthur

File: roles-in-tech/sales-engineering.md [hunk 4]
```diff
@@ -2,91 +2,100 @@
...
+- Guided Practice: 10 min
+- Independent Practice: 4 min
+- Check for Understanding: 10 min
 
 ### Prerequisites
 
-Here are links to lessons that should be completed before this lesson:
-
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
+NOne
```
Comment (alodahl) [inline roles-in-tech/sales-engineering.md]: NOne -> None

File: roles-in-tech/sales-engineering.md [hunk 5]
```diff
@@ -2,91 +2,100 @@
...
-- More things about the things
-	- This is a sub-thing about the things
-- Even more things about the things
-- Even more things about the things
+- Who is a sales engineer
+- Duties of a sales engineer
+- Traits of a good sales engineer
+- Steps to take for a career as a sales engineer
 
 ### Materials
```
Comment (alodahl) [inline roles-in-tech/sales-engineering.md]: Just to double check: 
1. Is everything that is assigned in Lesson, Practices or Challenge listed in the materials section?
2. Is there anything in the materials section that is not assigned, but is just additional?  If so, move it to supplemental Materials.
3. Is there anything in supplemental materials that is assigned? then move it to "materials".

**Pull request merged**